tests/client-dbus/Makefile

```diff
@@ -1,8 +1,13 @@
 TOX=tox
-
 .PHONY: lint
 lint:
 	$(TOX) -c tox.ini -e lint
 
 dbus-tests:
 	py.test-3 ./tests/dbus
+
```

Comment:
All the make targets are phony, so you might as well label them, like the lint target.

Reply:
Ok